Defs/ResearchProjectDefs/ResearchProjects_Assemblers.xml

```diff
@@ -73,15 +73,15 @@
 
   <ResearchProjectDef ParentName="PRF_AssemblersResearch">
     <defName>PRF_MagneticTape</defName>
-    <label>Magnetic Tape</label>
+    <label>magnetic tape</label>
     <baseCost>1000</baseCost>
     <prerequisites>
       <li>PRF_CoreTierII</li>
     </prerequisites>
     <researchViewX>6</researchViewX>
     <description>A specialized industrial tape used for recording audio, a useful method for archiving important instructions and general information.</description>
     <discoveredLetterTitle>About: Intro to recipe db.</discoveredLetterTitle>
-    <discoveredLetterText>You have learned how to make and use old-school magnetic tape to store instructions.\nMake data disks at the machining table.\n This is used to import recipes from any workbench into your recipe db.\n You do so by placing it near a workbench and clicking import on those recipes you want it to store.  Afterwards, you can move it to another workbench and copy more. Or you can move it to the Assembler you want to link to it.\n Recipes will be stored forever in your database, and you can safely move it around.  Please note that if you remove it from your assemblers, they will loose their configured bills.</discoveredLetterText>
+    <discoveredLetterText>You have learned how to make and use old-school magnetic tape to store instructions.\nMake data disks at the machining table.\nThis is used to import recipes from any workbench into your recipe db.\nYou do so by placing it near a workbench and clicking import on those recipes you want it to store. Afterward, you can move it to another workbench and copy more. Or you can move it to the Assembler you want to link to it.\nRecipes will be stored forever in your database, and you can safely move it around. Please note that if you remove it from your assemblers, they will lose their configured bills.</discoveredLetterText>
```

Comment:
"Afterward" is American English and "afterwards" British English. I just tried to avoid mixing and chose American (not sure if it was prevalent or not) which is also the default language for rimworld seems.

Reply:
checks out